### DISCUSSION TITLE
why cant install netcat?

### DISCUSSION THREAD

**Discussion by inhdyaii** [termux/termux-packages#7741] [Q&A] [answered]
I can nat install pkg netcat in termux V 117 if any one hav idea please help me

**Comment by ghost** [comment, marked as answer]
`netcat` has been renamed to `nmap-ncat` since being compiled from NMap source code. So you have 2 variants:
```
pkg install nmap-ncat
```

Or, if needed OpenBSD version of NetCat:
```
pkg install netcat-openbsd
```

Use `pkg search` to discover proper names of packages.
